osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -225,8 +244,8 @@ private Color4 getTriangleShade()
             {

```

Comment:
Just a comment on the side: We may want to drop transparent triangles within the regular triangles background as well. It looks to me like flyte's designs use opaque triangles in all samples (and I personally prefer their look).